packages/flutter_tools/templates/package/README.md.tmpl

```diff
@@ -1,14 +1,36 @@
-# {{projectName}}
+<!-- 
+This README describes the package. If you publish this package to pub.dev,
+then this README supplies most of the contents of the package page.
```

Comment:
There is a *lot* of other content (e.g. API docs).

Perhaps "this README will be shown as the landing page for your package on pub.dev"?

Reply:
> There is a _lot_ of other content (e.g. API docs).
> 
> Perhaps "this README will be shown as the landing page for your package on pub.dev"?

I don't think that quite works, since (as you said) there's a lot of other content on the landing page. I've suggested a change below.